### ISSUE TITLE
(3.1.0) Mods With Conflicts Always Utilize One Mod's Contents

### ISSUE THREAD

**Issue by SkyNinja** [NiceneNerd/BCML#86]
When installing mods with conflicting files in 3.0, the files of a specific mod always take precedence regardless of load order, whereas in 2.8, the higher load order properly took precedence.

E.G., [Linkle Mod](https://gamebanana.com/skins/163132) and [Linkle & Zelda RWBY Costume](https://gamebanana.com/skins/182338). When installing these together in 3.0+, the content from Linkle Mod will always display in-game, regardless of load order.

**Comment by GingerAvalanche** [comment]
Can you provide evidence for this? Screenshots of file contents, dates modified, or similar? I use the RWBY costume with Zelda's Ballad and it's not happening to me.

**Comment by NiceneNerd** [comment]
Have you tested whether this happens in more cases than that one specifically?

Anyway, I can't replicate it. I just installed those two mods, put RWBY at the higher priority, and it worked precisely as expected.

**Comment by SkyNinja** [comment]
This video should address both of your concerns. In it, I show the issue occurring with both the RWBY outfit, as well as with [LINKLE: Worn Old Clothes+Leather Sandals](https://gamebanana.com/skins/182189). The video is uncut from beginning to end, but some of the longer, more drawn-out segments have been sped up for the sake of time.
https://youtu.be/79dLuR0QmCM

There's also discussion of what seems to be the same issue in the comments of one of Sockpoppet's other mods: https://gamebanana.com/skins/180344

It's probably also worth noting that I'm on Windows 10, using the installer build of BCML.

**Comment by NiceneNerd** [comment]
It seems like you're using the wrong outfit. RWBY affects the Dark Set, not the Phantom Set.

**Comment by SkyNinja** [comment]
That's the Phantom Ganon set that I'm trying, provided by the "EX: Dark Armor" quest. The Phantom Armor is, in my video, the set that starts directly after the Midna helmet. Even disregarding that, the last segment of the video shows the same thing happening with the other mod, as well.

**Comment by NiceneNerd** [comment]
I'm still confused: what does the Phantom set have to do with anything?

As for the other one, some odd things can happen with certain BCML 2.8 BNPs, did you try installing the graphic pack version?

**Comment by SkyNinja** [comment]
It's for the Dark Link set? This game's messy naming conventions kill me.
As for the other outfit, I hadn't. I wasn't actually interested in it, it's just one that I came by to test with. I tried with a different mod for the same outfit, i2m's Old Outfit Replacement, and it worked. 
So, ultimately, I guess this was a compounding of outdated information, confusion based on the name of the Phantom Ganon quest, and a possibly buggy mod seeming to confirm the issue.
Wow. Sorry about wasting your time, and thanks.
